db/seeds.rb

```diff
@@ -1,3 +1,7 @@
+ActiveSupport.on_load(:action_mailer) do
+  Rails.application.reload_routes_unless_loaded
+end
```

Comment:
Even though emails don't actually get sent from the seeds, the mailers blow up if the routes aren't loaded.